contracts/StablecoinConverter.sol

```diff
@@ -240,5 +253,29 @@ contract StablecoinConverter is EpochTokenLocker {
             }
         }
     }
-}
 
+    function findPriceIndex(uint16 index, uint16[] memory tokenIdForPrice) private pure returns (uint) {
```

Comment:
This would probably be cleaner on a library or something that is not attached to pricesIndexes and potentially already written (usually if we find ourselves implementing binary search outside of a coding interview, we are doing something wrong).

If I understand https://solidity.readthedocs.io/en/latest/contracts.html#libraries correctly using a libraries internal function should not incur any gas overhead.

You cold then use something like

```
using BinarySeach for uint16[];
tokenIdForPrice.find(index)
```

which might be nicer to read than `findPriceIndex(index, tokenIdForPrice)`

Reply:
I completely agree that this would be a more readable way. 
However, in the next PR, I will change this find(index) function to a very custom implementation with index shifting due to the feeToken. 
Let's wait for the next PR and then we can take a look, whether this would still be combinable with a "standard implementation"